### DISCUSSION TITLE
How to disable mentions button and references button in MarkdownEditor.DefaultToolbarButtons?

### DISCUSSION THREAD

**Discussion by gavinliu6** [primer/react#2720] [Q&A] [answered]
I use MarkdownEditor in my project, and I don't need Inline suggestions: `@` mentions and `# `references. So how to disable/hide these two buttons in the toolbar?

**Comment by lesliecdubs** [comment, marked as answer]
Hi @gavinliu6 👋🏻 

The `MarkdownEditor` component is currently in draft/experimental stage (https://primer.style/contribute/component-lifecycle#experimental), which means it's the earliest stages of maturity. We don't currently have plans to make the inline suggestion functionality optional/hidden, although it's possible we may consider this in the future.

cc @justinbyo
